support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -349,13 +347,6 @@ export function ThreeTierLanding({
 	const selectedContributionRatePlan =
 		contributionType === 'ANNUAL' ? 'Annual' : 'Monthly';
 
-	const inNewsPaperArchiveBenefit = ['v1', 'v2'].includes(
```

Comment:
Would this need to be switched back on when the archives are  available?  It deosnt seem it in abtestDefinition mind

Reply:
Good point. I'll find out what's happening with that test.
V1 of that test would be fine to configure from the tool, but V2 involves some extra UI